### PR TITLE
Remove Ubuntu Trusty

### PR DESCRIPTION
**Pull request by blag**
Remove configuration for Ubuntu Trusty, since we dropped support for that after ST2 v3.1.

I'm not really sure what I'm doing here, so any help or direction would be more than welcome.

I did this work on top of the work for #84 as well, in the hopes that it makes this easier to test.

### CODE REVIEW

**Comment by blag** [comment]
Fixed.

File: packagingrunner/xenial/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM quay.io/dennybaa/droneruby:trusty-rbenv
+FROM quay.io/dennybaa/droneruby:xenial-rbenv
```
Comment (armab) [inline packagingrunner/xenial/Dockerfile:1]: This change is not needed and doesn't affect producing trusty packages.

```suggestion
FROM quay.io/dennybaa/droneruby:trusty-rbenv
```

File: packagingrunner/suite.yml [hunk 2]
```diff
@@ -1,2 +1,2 @@
 registry: quay.io/stackstorm/
-latest: trusty
+latest: xenial
```
Comment (armab) [inline packagingrunner/suite.yml:2]:
```suggestion
latest: trusty
```

File: packagingrunner/xenial/Dockerfile (hunk 1 above)
Comment (punkrokk) [reply packagingrunner/xenial/Dockerfile:1]: I agree

File: packagingrunner/suite.yml (hunk 2 above)
Comment (punkrokk) [reply packagingrunner/suite.yml:2]: ditto

**Comment by armab** [comment]
FYI just removed `trusty` from triggering Docker Hub automated builds.

**Comment by blag** [comment]
Incorporated the two suggested changes and resolved all issues.

**Comment by blag** [comment]
I'm not sure how to kick off a CI build again, but I'd like to before this is merged.

**Comment by blag** [comment]
@armab Resolved `packagingrunner/xenial/Dockerfile`.

**Pull request merged**